Modules/_testcapi/immortal.c

```diff
@@ -0,0 +1,103 @@
+#include "parts.h"
+
+static PyObject*
+test_immortal_bool(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    PyObject* objects[] = {Py_True, Py_False};
+    Py_ssize_t n = Py_ARRAY_LENGTH(objects);
+    for(Py_ssize_t i = 0; i < n; i++) {
+        PyObject* obj = objects[i];
+        if (!_Py_IsImmortal(obj)) {
+            PyErr_Format(PyExc_RuntimeError, "%R should be the immportal object.", obj);
+            return NULL;
+        }
```

Comment:
It's ok to use asserts in the test CAPI C tests.

```suggestion
        assert(_Py_IsImmortal(obj));
```